### PR TITLE
everyone sucks except me

### PR DESCRIPTION
**Pull request by ghost**
whatever fuckwit changed loyalty implants to mindshield implants can choke on a cock and die

they're loyalty implants again + some more fluff text and a sec buff vs gangs
#### Changelog

:cl:
rscadd:Mindshield implants are now called loyalty implants again.
rscadd:Loyalty implants now are not destroyed when implanting a gangster-you only need one instead of two now.
rscadd:Loyalty implants now have some more fluff messages on some antagonists.
rscadd:Those corrupted by false religions are now revealed to resist implants when implanted.
/:cl:

### CODE REVIEW

**Comment by X-TheDark** [comment]
Why? What's wrong with mindshield implants?

**Comment by Time-Green** [comment]
That's one way to stretch out the changelog.

**Comment by GrayRachnid** [comment]
no
you didn't even change back, you also buffed them
even more no.

**Comment by Alek2ander** [comment]
Loyalty implies people have to arrpee obedience, which literally never happens. So :-1:

**Comment by EMT321** [comment]
I can understand making it 1 implant to deconvert+loyalty for gang/rev, and making it conspicuously not appear for other antags- with no fluff text saying "X resisted the implant". This makes people implanting without sechuds less likely to nail a cultists/head rev/gangboss.

**Comment by ParagonPrime** [comment]
IMO mindshield implants are far, far more interesting for RP, not to mention with less chance of random lol-implanting - trying to poke everybody with an implant that makes you immune to mind control but doesn't guarantee loyalty to nanotrasen makes a lot less sense than stabbing every random motherfucker with a loyalty implant (theoretically you'd be justified in implanting everyone who enters the brig as a warden "just in case")

also why the fuck would you make a mindshield buff _without removing traitorsec loyalty implant fuckery_

traitor + mindshield at least sort of makes sense from an IC perspective (even though balance-wise it is broken as fuck), but traitor + loyalty implant makes zero sense

**Comment by Zostroll** [comment]
Not being able to trust anyone is a part of this game. The mindshield implant yields alot more opportunities for rp than the loyalty implant.

File: code/game/objects/items/weapons/implants/implant_loyality.dm [hunk 1]
```diff
@@ -22,38 +22,48 @@
 		var/obj/item/weapon/implant/mindslave/imp = locate(/obj/item/weapon/implant/mindslave) in target
 		if(imp)
 			imp.removed(target)
-		if((target.mind in (ticker.mode.head_revolutionaries | ticker.mode.get_gang_bosses())) || is_shadow_or_thrall(target))
-			target.visible_message("<span class='warning'>[target] seems to resist the implant!</span>", "<span class='warning'>You feel something interfering with your mental conditioning, but you resist it!</span>")
+		if((target.mind in (ticker.mode.head_revolutionaries | ticker.mode.get_gang_bosses())) || is_shadow_or_thrall(target) || (target.mind in ticker.mode.cult) || (target.mind in ticker.mode.blue_deity_prophets | ticker.mode.red_deity_prophets | ticker.mode.red_deity_followers | ticker.mode.blue_deity_followers))
```
Comment (ShadowDeath6) [inline code/game/objects/items/weapons/implants/implant_loyality.dm:25]: does cult include clock cult

**Comment by Super3222** [comment]
adam. from a players perspective, renaming the mindshield implants was a good idea. the crew shit themselves when they heard "the redshirts are coming with another lockbox!" because it's literally mind control. but 'mind shield' is the opposite of that. it sets up NT as protagonists rather than than antagonists that the game sets up. it's a bit of a contradiction and, somewhat bothered me when I use to non-stop play sec.

also this changes the meta of the game slightly, considering this'll give sec the upperhand on identifying cultists. "crazy looking book & was running around the hallways naked? take em to the back for an implant". so really, i don't know how i feel about that one.

**Comment by ParagonPrime** [comment]
> Not being able to trust anyone is a part of this game. The mindshield implant yields alot more opportunities for rp than the loyalty implant.

Yes, only exception being Sec should be able to trust Sec. AKA, end traitorsec for fucking ever by making them resist implanting.

Lings should be able to keep it, but maybe it decays over time or something to keep it balanced and encourage you to not just stay in the one form all the time.

**Comment by BOBsHERE** [comment]
Don't let implants identify cultists because then every round is just "he might be an antag? better implant to confirm." Right now its a good thing that idiot security players are implanting and releasing cultists without holy watering, as much as i hate it when it happens. The game is based on knowledge=power, so the more things that require a knowledge check the better.

**Comment by ParagonPrime** [comment]
loyalty implant would make IC sense to give to everyone, thus it is shit

mindshield implant, you'd have no IC way of knowing that it could identify antags and no IC reason to give it unless cult/rev/gang is already CONFIRMED INDEPENDENTLY, thus it's far more meta to give to random prisoners 4noraisin

**Comment by Notamaniac** [comment]
The main reason loyalty implants were refluffed was to sort out that grey area when caught tators or grey tiding crew were implanted and make it a lot less ambiguous to whether could still break space law. Bomb collars were given as the alternative to keep people in line. Fluffwise this PR just serves to bring up that ambiguity again.

I don't really agree with the implant buffs either, usually either sec gets steam rolled, or they manage to secure cargo so they can order more. All this does is speed up the case in the latter, unless sec get bombed in cargo, they're probably going to be able to defeat whatever team antag there is.
Making culties resist implants also stops them from infiltrating sec and I don't see what benefit that really adds; paranoia is a main part of SS13 and this seeks to remove that

**Comment by ParagonPrime** [comment]
sec should NEVER be able to be "infiltrated" except by Changelings because that's their fucking job and a central part of their game mechanics, even if it does need a slight nerf

the paranoia and whatnot would be nice- if there were any. currently antagsec = "grab a taser, wait until you're alone with another sec member, stun, cuff, murder" with zero RP or "paranoia". it's giving some of the most powerful weapons _in the game_ (no, seriously, if you have both a stunbaton and disabler, you can basically 1v1 take on any humanoid antag short of a shadowling) to antags for very little effort and that should absolutely not be the case

buffing the implants to allow one-step deconvert-and-implant for gang/rev is good, that's the only part of this PR i actually like

File: code/game/objects/items/weapons/implants/implant_loyality.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/weapons/implants/implant_loyality.dm:25]: yes

**Comment by ghost** [comment]
it just fucking changes a name, you mongs

that's all it does

stop throwing a fit

here's a reason.

Because 1. NT are _not the good guys_ by any means. They give absolutely no shits about their employees, SS13 is literally where they send the murderous crackpots to die, there are no safety features almost anywhere-The syndicate are the bad guys, but NT is morally grey at best.
1. Mindshield implanted crew getting implanted and then immediately going to medbay for implant removal and conversion should be bannable-and it was, under loyalty implants, but under mindshield there's no reason for people to resist conversion to go kill asshole sec force implanting you. Not to mention 99% of people are implanted and then don't help security whatosever because they're buttmad about being implanted.
2. There was no ambiguity with loyalty implants. If you are implanted as a regular crewmember, you cannot disobey security or the Captain unless they're acting against NT's interests. You can't commit crimes unless it's acting in NT's interest. Et cetera. Mindshield implants **had** that ambiguity-you could get implanted and start disarming security immediately after, because despite being deconverted you're still hyper buttmad that sec arrested and force implanted you, and since it's not mind control anymore, you have the right to attack them for forcibly putting a chip in your brain.  Hell, even if you're nonantag and sec forcibly detains, bucklecuffs you, and comes back 20 minutes later to implant you, you get up and start disarming sec and that's fine, because they were a bag of dicks to you and it's not mind control.

Loyalty implants physically prevented you from attacking security because you are loyal to NT-If you are arrested and implanted, nope, you can't be buttmad and harass sec because they were dicks. Glory to NT.
1. Mindshield implants as they currently exist make no sense. There's no reason not to go around implanting people at roundstart with them, because they literally just stop people from being turned into bad guys against NT, and so why would you not do that? Why would anyone in their right mind refuse the opportunity to avoid being mentally enslaved by a blood cult or crime syndicate and forced to do their will? Loyalty implants add back the whole 'morally questionable' thing, which is A. A key part of NT and B. A reason not to go and order a fuckton of implants and jab them up people's assholes beyond 'it's powergame though' and even THAT is debatable, for the above reasons I provided.

Mindshield implants, logically, can be handed out like candy, since there are no negatives and literally prevent you from being mentally enslaved. Loyalty implants ARE mental enslavement, and as a result can't just be thrown out freely-crew are stuck between morally ambiguous NT, preventing you from being converted to the syndicate at the cost of your free will, or the antagonists, forcing you to kill the heads/be part of a blood cult/summon a broken machine god/join a crime syndicate and kill people, but is that really any worse than what NT is doing to you?

So yes, there are several reasons.
1. Sec needed the buffs against gangs. Loyalty implants are expensive enough-all implants being destroyed on implanting gangsters mean is that you might as well just beat the guy to death instead of wasting 2 implants on deconversion. Which means more body count, which means more salty ghosts dead to security. We had it this way BEFORE the rebase-Where it took one implant-and it was fine as was.
2. Cult silently resisting loyalty implants makes no sense, seeing as gangsters, revs, resist them openly, and shadowlings and thralls (which have ABSOLUTELY NO RELATION TO THE SYNDICATE WHATSOEVER) also openly resist implants. If Shadowlings resist implants openly despite being an alien race that uses mental enslavement, why would a cult that uses mental enslavement also not demonstrate open resistance to the implant? Why is it silent? Not to mention, everyone already knows that cultists need to be holy watered to be deconverted-Loyalty implanting cultists and letting them join security is the sign of idiot security to start with, so it really has no balance impacts whatsoever, simply because if sec is dumb enough to implant-and-release cultists, they're probably too stupid to not die in the first 5 minutes anyway.

And before you comment 'cultists joining security lol' let me remind you that  **antag sec is AIDS and disabled for a reason.** Antag security is literally the worst, as almost anyone can testify-you're free of suspicion up until they catch you with an esword, and any calls of 'SEC ROGUE' are usually laughed at. And you can usually just play it off as being shitcurity and apologizing for it and get away with it scot free. Cultists joining security is obnoxious as shit-join sec, get implanted, grab a tazer, taze security, sacrifice, shuttle call. Real fun, isn't it? Except it's not. It's a steamroll strategy and hardly fun at all.

If loyalty implants make you loyal to NT, then why do cultists not demonstrate open resistance to it when implanted if shadowlings do?

Not to mention, Rat'Var and Cult needed the nerfs at the least (this barely counts as a nerf since everyone knows cultists need holy water to deconvert.) cult alone already has a stupidly high winrate if any of it's members are intelligent and know what it's doing and Rat'Var steamrolls HARD if left alone for more than 10 minutes and undiscovered. And again, this is **barely a nerf.**

The only ones who should logically not demonstrate resistance to implants are traitors-who are agents trained to infiltrate and thus should be able to resist silently and changelings-who are just fucking weird and alien.
1. @puremanbird Rounds **will not** become that and you know it. Why? Because it's already 'he might be an antag better implant to confirm', seeing as revs shadowlings and gangsters resist, and there's no reason NOT to implant people with mindshields as security seeing as they have no negative downsides and prevent people from being mentally enslaved against their will. There's no reason to refuse it as a normal crewmember either-And before you complain 'but it's a painful injection' nobody gives a fuck about pain or RPs it anyway so don't start.

In fact, rounds will have less 'better implant to check' simply because loyalty implants are MENTAL ENSLAVEMENT, and so doing so to someone 'just to check' or 'precautions in case of' is not only firmly morally BLACK and is liable to get you arrested or lynched, will also get you FUCKING BANNED, as you were before the rebase if you implanted people without a reason at all 'just in case' or 'he disarmed me a bunch, he might be a rev'. Because now, if someone disarms you a ton, and you mindshield implant, it  makes sense-Centcomm reports mention possible revs, someone's disarming you nonlethally, it might just be better to check and even if your assumption is wrong there's no harm in it. With this change, it's 'someone disarmed me a bunch, better ENSLAVE HIS MIND TO THE COMPANY.'

Meaning it's gonna happen less, both ICly and due to admin enforcement, just as it was during the rebase. 
1. @puremanbird JUST TO CUT YOU OFF BEFORE IT STARTS, if you're about to say 'but then sec will implant to check if someone is a cultist!!!!!' that's hardly an impediment. If they think you're a cultist, they'll bucklecuff you to a bed anyway and force feed you holy water regardless of implant or not. At least this reduces the amount of people force fed holy water for 20 minutes while nonantag on pure suspicion-which is completely fine from an admin POV, because 'they think you're a cultist, so they are allowed to give you holy water.' This means that there's less of that. 

AND since implants will be destroyed when they implant a cultist when they resist it, what that means is three things.

A. Security has to choose between wasting an implant to confirm if someone's a cultist or not, or holy watering him which takes longer but is basically free and will deconvert the guy instead of implant THEN holy water.

B. It removes the annoying tactic of implant a suspected cultist then holy water, which means when deconverted they're instantly unable to be converted again because lol, implant. Now you have to deconvert, and then implant. This can be considered a buff to counteract-

C. The fact that cult antag sec, which is universally agreed AIDS, is removed, as i've mentioned above. It's barely a nerf if you're thinking of it in terms of 'idiot sec implants a guy and lets him go and then he goe and summons nar'sie' and also prevents the god awful tactic of 'join sec taze convert slap each other's asses'.

@ParagonPrime As I have mentioned above, Mindshield implant has no reason NOT to hand out like candy. You can give it out to everyone to no downside and also preventing people from being converted by possible antagonists mentioned on the CC report-and with that as your reason, you're untouchable by the admin team, because 'CC report says this might be a threat, so I might as well take preventative action, which is what people would do if they're not autistic.' 

Loyalty implant HAS a reason to not hand out to everyone because it's MENTAL ENSLAVEMENT. Meaning, Paragon, it's literally the **reverse** of what you said above.

@Super3222 Cult right now has literally no metashield at all except for like the barest example. If someone has a crazy looking book, it's automatic taze arrest call the chaplain and start mass holy watering. This doesn't give them the upper hand on identifying cultists, especially because AS ABOVE LIKE I SAID the implant is destroyed on a failed cult implant, meaning it's a huge waste of time, and if you catch someone with a crazy book they're getting deconv'd anyway.

@Alek2ander They have to , or they get banned. So :+1: 

@Zostroll That's a bullshit reason and you know it.

## I have adressed literally everyone's points and comments in the above post. If you respond saying something that I already adressed above, i'm going to get really mad, because you didn't read the above post.

**Comment by Super3222** [comment]
Adam roughly 90% of your post was bullshit and, I'll explain why tomorrow.

**Comment by ParagonPrime** [comment]
Adam, if NT doesn't give a fuck about mental enslavement, THE SECURITY CREW OF AN NT STATION WOULDN'T GIVE A FUCK ABOUT THE MORALS OF MENTAL ENSLAVEMENT EITHER

ICly there's even _more_ reason to distribute loyalty implants

but yknow what, here's an idea

why don't we get multiple tiers of implant?

mindshield and loyalty

the former just deconverts you and makes you immune to future conversion

the latter forces you to be a literal slave to nanotrasen and announces it very boldly in huge red text

obviously the latter is harder to acquire and costs more in cargo

also, no, traitors should 100% resist implants

maybe stealthresisting is fine to punish unobservant sec, but they should not be able to take an implant and then just be fine with slaughtering all of sec

what would be even funnier is if they got loyalty implanted and started dumping the eswords and revolvers from their pockets

**Comment by ShadowDeath6** [comment]
make it so that every antag other than traitor resists the implant just to make paragon more assmad

**Comment by ParagonPrime** [comment]
shadow, if you really don't see what's wrong with traitorsec you don't deserve to be a coder

or play ss13

i'm dead serious

**Comment by BOBsHERE** [comment]
Okay, im sold. I just dont want stupid security implanting normal traitors and giving them free pass to do everything because LOL implant.

**Comment by Super3222** [comment]
Did I say tomorrow? I meant when I get the time.

**Comment by ghost** [comment]
@Super3222 it's tomorrow

**Comment by Super3222** [comment]
@AdamElTablawy What I meant was tomorrows tomorrow. You'll definitely hear from me tomorrow.

**Comment by Super3222** [comment]
> 1. NT are not the good guys by any means. They give absolutely no shits about their employees, SS13 is literally where they send the murderous crackpots to die, there are no safety features almost anywhere-The syndicate are the bad guys, but NT is morally grey at best.

Yes, NT is shit, NT is suppose to be the "don't give a fuck about you guys but you work for me haha", NT is suppose to be this big mysterious company which we barely know about in the game until someone scribles in a few extra points of lore every now and then but, changing mindshield back to loyalty is going backwards. Because things like this are already set in stone and people enjoy it. I haven't seen one single person complain about the concept at all BUT YOU. Maybe you're doing it because you want people to have an excuse to stop punching officers but, you forget being administered a loyalty implant shouldn't change people from being stupid greytiders. Admins should because, it's against our rules and if you freak out after being mindshielded. There's a line that differentiates the two. An implant is not going to change peoples personalities. The way mindshield implants are set up right now is fine, and makes fucking sense rather than "WATCH OUT, MY MIND CONTROLS BETTER THAN YOURS ANTAGONISTS!". Like, what even.

If you're looking for an excuse to ban someone for being obnoxious and, fucking with security's job to stop a group antag, then don't look at mindshield implants.
Because being protected from mind conversion, whether forced or not, doesn't give someone permission to be a fucknut.

Besides, my understanding when I first starting playing was 'SS13' was a low-budget plasma research facility which was owned by NT who did care sometimes and didn't care at other times.
Obviously, that ideas been joggeled around, cut open, splashed with hot boiling water, dried, nurtured, and then had it's skin peeled off.
What I'm getting at is the lore behind NT can never truthfully be set unless you're on an RP server that cares. And Yogstation does not care.

> There was no ambiguity with loyalty implants. If you are implanted as a regular crewmember, you cannot disobey security or the Captain unless they're acting against NT's interests. You can't commit crimes unless it's acting in NT's interest. Et cetera. Mindshield implants had that ambiguity-you could get implanted and start disarming security immediately after, because despite being deconverted you're still hyper buttmad that sec arrested and force implanted you, and since it's not mind control anymore, you have the right to attack them for forcibly putting a chip in your brain. Hell, even if you're nonantag and sec forcibly detains, bucklecuffs you, and comes back 20 minutes later to implant you, you get up and start disarming sec and that's fine, because they were a bag of dicks to you and it's not mind control.

But there's literally no fucking point to this. And while reading this, recap what I said in my last point because that applies too...

You want a prisoner to be more lawful? Implant them.
Get called out for powergaming because they weren't a sus enemy of the cooperation.

Successfully deconvert an antag, they get pissed off, start disarming the shit out of you?
Protect yourself.

Or you know, if you're really looking out for the boys in red all it takes is one simple, "You are no longr an antagonist, so stop liking it" from the gods.

And I know what I'm talking about because I've been through it after the rebase.
3 or 4 things happen after deconverting & implanting a guy:

A) Flat out suicide and, I take their loot.
B) Start running around greytiding and, get their ass beat and thrown out for it. Come back in again? Perma, I usually prefer the insanity ward because as I say it "their minds are beyond saving, not even a specialized NT anti-mind control implant was able to cure them". Else, I take my sweet time to torment them by throwing them into the gulag. At that point, a normal prisoners instinct is to run across the lava to the other side. Which is why I always pack some popcorn before every trip.
C) Actually help out security. Gear up, load up, and help take back the station.

ooor D) Just walk away.

I appreciate the loyalty cuddling but, we don't need it. Some people may want to become antagonists during shit like clockwork, cult, rev, or gang or some people may not. But who cares, he's rightful to throw a big fuss about it, but hell, he'll get taken down for it unless the officers are BAD.

Besides, if they run back to the enemy cooperation for a conversion, that's breaking a capital law purposefully which is a direct violation of Yogs rules given the circumstances.

> Loyalty implants physically prevented you from attacking security because you are loyal to NT-If you are arrested and implanted, nope, you can't be buttmad and harass sec because they were dicks. Glory to NT.
> 
> Mindshield implants as they currently exist make no sense. There's no reason not to go around implanting people at roundstart with them, because they literally just stop people from being turned into bad guys against NT, and so why would you not do that? Why would anyone in their right mind refuse the opportunity to avoid being mentally enslaved by a blood cult or crime syndicate and forced to do their will? Loyalty implants add back the whole 'morally questionable' thing, which is A. A key part of NT and B. A reason not to go and order a fuckton of implants and jab them up people's assholes beyond 'it's powergame though' and even THAT is debatable, for the above reasons I provided.

One, no loyalty implants didn't. Whether you're talking about ingame mechanics (probably not) or actually roleplay in the game. Because they really didn't.

And my God, mindshield implants do make sense. It's an implant for emergency situations where terrorist organizations are trying to take over the station. They aren't fucking lolipops you can throw here and there. They are FOR EMERGENCIES. Security is only authorized to distribute them THROUGH EMERGENCY SITUATIONS, the same way LOYALTY IMPLANTS use to be handled. So get that one through your head and quit playing with words here.

Now off to your second point. It's up to the goddamn players to decide what they want to do in the game. Do you want nanochips in your brain to protect you from mindcontrol? Do you not want one? It's their own characters decision to follow up with that. If it was a fucking problem Adam, everyone on board would be flooding to the brig to get an implant. But they don't and they won't ever unless a group antag goes loud and they wanna kick ass.

And why do they not want to do that in the first place? Because people just want to fucking play the game for what it is. Security has implants, crew members don't, antagonists do not want implants. Crew members don't want fucking implants because they don't want to be forced into decisions like that. It's really quite simple and, you're just overcomplicating easy to get shit. No one's going around massimplanting everyone during a traitor round, less a blob round. And if an admin does let that go, then you're nothing but a shitmin that's letting some random greytider-in-a-sec-position fuck with everyone elses game.

And besides, another fucking thing we have admins for. Wew lad. It's almost as if **abusing mechanics in the game to mess with people is prohibited by admins**.
You're talking about this shit as if you're looping through everything like it's all AI laws, which I'm solefully glad we're not.

> Sec needed the buffs against gangs. Loyalty implants are expensive enough-all implants being destroyed on implanting gangsters mean is that you might as well just beat the guy to death instead of wasting 2 implants on deconversion. Which means more body count, which means more salty ghosts dead to security. We had it this way BEFORE the rebase-Where it took one implant-and it was fine as was.

This piece is actually rational and I can get behind. Loyalty implants are expensive and, I know someone's going to bring up the goof-cargo situation. But, that's just another factor in SS13 currently and you can't always wager on "oh boy a cargo tech? we're going to have over a million points!!!". And I've already made this point in another PR, multiple ways you can rack up points but, with how broken Cargo is now I don't wanna get into it again.

> Cult silently resisting loyalty implants makes no sense, seeing as gangsters, revs, resist them openly, and shadowlings and thralls (which have ABSOLUTELY NO RELATION TO THE SYNDICATE WHATSOEVER) also openly resist implants.

I agree with your point here but, I want to make something very clear to you Adam, because I believe it went over your head.

Because it's apart of their fucking gamemode, Adam. It's a built in mechanic, and I'm sure they've talked about this lots over on /tg/.

Gangsters, revs, and whoever the fuck are raised on a common tier of leaders -> underlings. Specific syndicate agents are more specialized, and secretive. They aren't respondent to implants within good reason, because abusing mindshield/loyalty implants for that purpose would be a bad habit in the game & mode in general. I'm sure you already know this.
Then you have the shadowlings, who I don't doubt resist implants because that's just the nature of their race. Besides, they use to be able to break implants until we "balanced" them (which by the way, even breaches the original lore Xhuis made up). So I can't say so much on that piece but, a shadowling that waits for that long and gets an implant DESERVES to get caught.

You might as well ask, "hey mate, why the hell can clockwork cultists convert implanted people but narisian dogs can't!".

Own. Gamemode.
It's how they work, and they do work well that way.

And the Chaplain was designed to be the one to signify cultists. That's another interesting thing about the gamemode. The struggle between keeping the Chap alive, because having them alive completely goes against cults.

There already are implications of how to shutdown a cult without claiming a cult. Purely by permabrigging people connected to a crime aside from "CULT!". Like, how a majority of people were recently brigged each carrying an extremely shady textbook. This is a technique that I respectively use in the game while the Chaplain is unavaliable.

> And before you comment 'cultists joining security lol' let me remind you that antag sec is AIDS and disabled for a reason. Antag security is literally the worst, as almost anyone can testify-you're free of suspicion up until they catch you with an esword, and any calls of 'SEC ROGUE' are usually laughed at. And you can usually just play it off as being shitcurity and apologizing for it and get away with it scot free. Cultists joining security is obnoxious as shit-join sec, get implanted, grab a tazer, taze security, sacrifice, shuttle call. Real fun, isn't it? Except it's not. It's a steamroll strategy and hardly fun at all.

Which I completely agree on with you but, it's funny hearing this from the same guy that was praising Traitor Det earlier. And don't even fucking pull up the arguement, 'detective ain't apart of sec' because he fucking is. Otherwise, he'd be implantless and foresinic scanners would be in the sec vendors.

So Sec Cult should definately go out the window. This change is needed.

> In fact, rounds will have less 'better implant to check' simply because loyalty implants are MENTAL ENSLAVEMENT, and so doing so to someone 'just to check' or 'precautions in case of' is not only firmly morally BLACK and is liable to get you arrested or lynched, will also get you FUCKING BANNED, as you were before the rebase if you implanted people without a reason at all 'just in case' or 'he disarmed me a bunch, he might be a rev'. Because now, if someone disarms you a ton, and you mindshield implant, it makes sense-Centcomm reports mention possible revs, someone's disarming you nonlethally, it might just be better to check and even if your assumption is wrong there's no harm in it. With this change, it's 'someone disarmed me a bunch, better ENSLAVE HIS MIND TO THE COMPANY.'

This is a big dumb lie because with that logic you're just letting someone get away with with abusing a **re-lored** mechanic, which hasn't changed GAME-PLAY WISE.

The use and handling has stayed the same, Adam. It's just how it dissolved into the game is different. And it's clear common sense.

If someone is disarming you, then you permabrig them.
There's no reason to bust out the mindshield implants because they are not for situations like that.
And unfortunately, I'm going to have to repeat myself again. Mindshield implants are not lolipops people can freely distribute. Just like loyalty implants, you should only be administering them when the situation calls for it and you have appropriate proof.

Someone randomly disarming you? Not proof.
Seeing someone flash someone else w/ a revtool. That's proof.

If you let Security mass implant people because of Exhibit A, than I repeat, you are a shitmin. That's not proof, that's a simple crime which calls for a 'briggin.

> Loyalty implant HAS a reason to not hand out to everyone because it's MENTAL ENSLAVEMENT. Meaning, Paragon, it's literally the reverse of what you said above.

As someone who I don't see regularly playing security, I don't think you really understand.
Loyalty implants aren't administered, not because it's "mental enslavement" but, because they are armory equipment which should only be used to counter enemies of the cooperation and free the mind of employees.

Or in the loyalty implants case, "make them 100x more loyal".

So in conclusion...
Don't send Security back into the caveman era.

Extra:
And No Paragom, you don't need to compromise with this shit. You're basically adding another mindslave implant, but NT brand. The idea of any enslavement is just a goofy trend.

**Comment by ghost** [comment]
before i shit all over your next post i'd like to congratulate you for making a giant post the likes of which impresses me

i'll contradict most of what you said tomorrow but a few things
1. i was probably buzzed when i liked antag det because it was ass and nobody would use traitor det to like frame people for their crimes they just shot all of sec and took the armory
2. for the most part everything above is like 'well that's why we have admins' hello yes i am an admin i am aware of this i'm literally on for 90% of the fucking day and i'm getting tired for the sake of my own sanity please help
3. i'm not saying i'd let sec get away with lol mass implants for everyone i mean that it results in a lot of argumentative chucklefucks coming in and fucking whining about 'no i should be able to do this it's RP it makes sense !!!! XX!!!!! your a shitmin and killing my special snowflake rohelplayyyy!!!!XXXXXDDD!!' and it's fucking obnoxious to have to go through a thirty minute ticket with people who absolutely refuse to accept that it's treated as 'ar pee' in the same way as 'lol im psycho' is roleplay

and before you say 'well slap them with a ban if they don't listen' i can't just immediately ban someone for not instantly agreeing with everything i say, i have to go through the motions and red tape so that i can establish they aren't going to listen  and keep slapping their dicks about 'but its my ROLEPLAY' despite the fact i know they aren't going to listen anyway and time would be better spent banning them immediately

w/e i'll finish up your post tommorow

**Comment by ghost** [comment]
and yes super admittedly most of those points were based on arrr peee, not game mechanics

game mechanics wise it literally changes nothing because it's literally just a name rename

there's no negative or positive impact aside from 'muh lore and roleplay' 

the way thigns are actually treated (attacking sec when mindshielded or implanted) still gets you a ban anyway regardless of what they're called, it just makes for unnecessary confusion among newbies and newmins

**Comment by ParagonPrime** [comment]
yeah except there ARE a LOT of scenarios in which the difference is very evident

let's say that a Captain well within space law and obedient to NanoTrasen bans wearing masks

loyalty implant: "CAPTAIN SAID IT AND HE ISN'T BREAKING THE LAW OR WORKING AGAINST NT THUS I MUST OBEY FOR I AM SLAVE"

mindshield: "lolwut no"

**Comment by Dagdammit** [comment]
Refluffling loyalty implants to mindshield implants is (was) honestly my favorite tweak I've ever seen in the Yog patch notes. Anytime I'd be reading another server's wiki and see mention of loyalty implants, I'd reflect for a moment on how much better the "mindshield implants" option was.

Not 100% confident I can articulate why, but the core factors might be...
1: It was a much better fit for what the implants DO. Calling them loyalty implants raised all kinds of questions. "Wait, what conditions under space law call for the use of loyalty implantation? Am I like an AI with one law overriding another directly conflicting one? What are the mental mechanisms of this loyalty enforcement? Is it subtle or an overt thought override? There's like half a dozen ways to interpret this and I don't know which I'm supposed to roleplay..." Mindshield implants are immediately clear in terms of mechanism and function. "Oh cool, they block mind control. Oh, they might or might not undo brainwashing that happened pre-implant."

2: Fluff wise, it's just so much more REASONABLE. I don't want NanoTrasen to be the "good guys", but they generally aren't orwellian hyper-controlling tyrants. If they were, the "corporate" lawset would have a bunch of heavy-handed laws like "Treason must be punished without mercy" and "Sedition is the highest form of treason." Instead, the corporate lawset is both 0% more kind and 0% more evil than is necessary to safeguard NT's bottom line, while allowing massive freedom of action. The higher-ups are rational, amoral, profit-driven dicks. You may not LIKE being stuck on a station with a serious uncontained biohazard, but that doesn't mean the policy to withold the escape shuttle isn't the objectively smart course of action. It doesn't just happen "because NT is evil".

(And if amoral profit-driven dicks who were pretty smart wanted to mess with mind controlling their own people? Then Mindshield is the label they'd put on the box.)

Re: "There's no reason not to dispense mindshield implants." 
There's also no reason not to fortify escape, install better valves for atmos, or add useful scripts to telecomms? As a botanist I stick Omnizine oranges on the medbay front desk and give bottles of Strange Reagent to the paramedics. Making the station more efficient, effective and secure is something many roles pursue every round. Maybe they'll succeed. So long as sec only starts with 3-6 implants and they cost a decent amount for Cargo to order, I don't see the issue. Choosing whom to implant (and how many of them to use preemptively) seems like a pretty interesting call for an HoS/captain to make.

**Comment by ParagonPrime** [comment]
Dag, I agree with everything there but the last bit-

Fortifying escape, replacing valves (with inefficient, non-AI controllable ones) and distributing implants without a very good suspicion is metagame, pure and simple.

And you could argue that fortifying/replacing valves isn't metagame (and I'd disagree), but I doubt there's anyone who disagrees that randomly handing out expensive, fancy implants without cause is very meta.

**Comment by Dagdammit** [comment]
Fair enough, and I'm fine with that being the case. Couple comments in different directions:

-If you want to shore up the "mindshield != candy" justification, say they last about a week.
-I thought the whole point of that weird notice with 6 different possible threats the captain gets every single round start was to justify minimal counter-antag measures, like raising alert level. Is there some other point to that?

By the way, I'd be totally down for there being a "Grade II Mindshield Implant" that actually gave someone a silicon-style law about loyalty to Nanotrasen. Maybe have 1 per box and the station spawns with one box in the vault or captain/armory safe.

**Comment by ParagonPrime** [comment]
Minimal countermeasures =/= handing out expensive implants that are likely to be entirely useless and time-wasting if the threat isn't some mind-controllers

**Comment by ParagonPrime** [comment]
have we

just made

_adam_

_ragequit?_

**Comment by Super3222** [comment]
Feel free to make another PR again, minus the loyalty part of course.
